style.css

```diff
@@ -0,0 +1,233 @@
+html {
+    background-color: #fbf5eb;
+    font-family: 'Kumbh Sans';
+}
+
+.header {
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+    margin-left: auto;
+    margin-right: auto;
+}
+
+
+/*Media query for very big screens to make sure the title and search box works well together*/
+@media (min-width: 601px){
+    .header {
+        justify-content: space-between;
+    }
+
+    h1 {
+        padding-left: 30px;
+        padding-top: 30px;  
+    }
+
+    .searchbox {
+        float: right;
+        margin-top: 20px;
+        padding-right: 50px;
+    }
+}
+
+
+/*Media query for big screens to make sure the title and search box works well together*/
+@media (min-width: 533px) AND (max-width: 600px){
+    .header {
+        justify-content: space-between;
+    }
+
+    h1 {
+        padding-left: 30px;
+        padding-top: 30px;  
+    }
+
+    .searchbox {
+        float: right;
+        margin-top: 20px;
+        padding-right: 30px;
+    }
+}
+
+
+/*Media query for medium screens to make sure the title and search box works well together*/
+@media (min-width: 450px) AND (max-width: 532px){
+    .header {
+        padding-left: 40px;
+        padding-right: 40px;
+    }
+}
+
+
+h1 {
+    padding-right:auto;
+    padding-left: auto;
+    text-align:center;
+    font-size: 40px;
+    color:#00AFB9;
+}
+
+#search-input {
+    border-radius: 30px;
+    border: none;
+    padding: 8px 8px 8px 25px;
+    font-family: 'Kumbh Sans';
+}
+
+.wrapper {
+    display: flex;
+}
+
+#filter {
+    padding-right: 40px;
+    padding-left: 30px;
+}
+
+.filter-btn {
+    border-radius: 30px;
+    background-color:#ffe5b3;
+    color: #b42465;
+    padding: 10px;
+    margin: 3px;
+    border: none;
+    margin-top: -40px;
+    font-family: 'Kumbh Sans';
+    font-weight: bold;
+    font-size: 11px;
+    margin-bottom: 13px;
+    box-shadow: 2px 2px 2px rgba(0, 162, 255, 0.2);
+}
+
+.sort-btn {
+    border-radius: 30px;
+    background-color:#ffe5b3;
+    color: #b42465;
+    padding: 10px;
+    margin: 3px;
+    border: none;
+    margin-top: -40px;
+    font-family: 'Kumbh Sans';
+    font-weight: bold;
+    font-size: 11px;
+    margin-bottom: 13px;
+    box-shadow: 2px 2px 2px #00a2ff33;
+}
+
+button:hover {
```

Comment:
When you apply an effect to the button, like in this case the transition, is making all the UI around move, and this is to be avoided. A solution would be to wrap the button in a div and give the div a fixed height 👍